asab/web/container.py

```diff
@@ -50,7 +50,7 @@ class WebContainer(ConfigObject):
 		'rootdir': '',
 		'servertokens': 'full',  # Controls whether 'Server' response header field is included ('full') or faked 'prod' ()
 		'cors': '',
-		'preflightpaths':'/openidconnect/*, /test/*',
+		'preflightpaths': '/openidconnect/*, /test/*',
```

Comment:
`cors_preflight_paths`, also allow multiline (see e.g. seacat pki for that bit).

Reply:
How did you resolve this request?

Reply:
I allowed multiline and forgot to rename...